extensions/amp-access-laterpay/0.1/validator-amp-access-laterpay.protoascii

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-access-laterpay
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-access-laterpay extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
+  also_requires_tag: "amp-access extension .js script"
```

Comment:
Well. The dependency chain is `amp-access-layerpay -> amp-access -> amp-analytics`. Should I expand transitive dependencies here? Also, we're trying to remove `amp-access -> amp-analytics` dependency.